kythe/go/platform/tools/extraction/repotester.go

```diff
@@ -46,6 +46,7 @@ var (
 	repos      = flag.String("repos", "", "A comma delimited list of repos to test")
 	reposFile  = flag.String("repo_list_file", "", "A file that contains a newline delimited list of repos to test")
 	configPath = flag.String("config", "", "An optional config file to specify kythe.proto.ExtractionConfiguration logic")
+	index      = flag.Bool("index", false, "Whether to attempt to index a repo after successful extraction")
```

Comment:
Is this related to this PR?

Reply:
ah damnit I pulled in the wrong stuff.  Thanks for catching that.